index.js

```diff
@@ -35,27 +33,27 @@ const Home = () => {
   const [file, setFile] = react.useState({});
   const [fetchError, setFetchError] = react.useState(false);
 
-  // Whenever the URL changes then
-  // 1. Resolve the unpkg url for the request url
-  // 2. Fetch the package.json for the requested package
-  // 3. Fetch the /?meta for the requested package
-
-  /* Runs once and subscribes to url changes */
+  // Runs once and subscribes to url changes
   react.useEffect(() => {
     console.log('Setting up URL listener');
-    /* Rerender the app when pushState or replaceState are called */
-    ['pushState', 'replaceState'].map(event => {
+    // Rerender the app when pushState is called
+    ['pushState'].map(event => {
```

Comment:
Since we only have `'pushState'` in the array now, do we still need a `map` (`forEach`)? Or might `'replaceState'` need to be added back in down the road?

Reply:
To clarify, I'm suggesting removing the `map` and replacing instances of `event` with `'pushState'`

Reply:
It might need to be added back in the future, so I'm happy to leave it like this for now!